pkg/syncer/cnsoperator/controller/cnsnodevmattachment/cnsnodevmattachment_controller.go

```diff
@@ -525,29 +533,30 @@ func removeFinalizerFromCRDInstance(ctx context.Context,
 }
 
 // isVmCrPresent checks whether VM CR is present in SV namespace
-// with given vmuuid.
+// with given vmuuid and returns true or false along with the
+// VirtualMachine CR object if it is found
 func isVmCrPresent(ctx context.Context, vmOperatorClient client.Client,
-	vmuuid string, namespace string) (bool, error) {
+	vmuuid string, namespace string) (bool, *vmoperatortypes.VirtualMachine, error) {
```

Comment:
We can remove `bool` from the return values.

We can just return the `VirtualMachine` object when we find the VM, and nil in other case. The caller can check whether the object is nil or not.
Let me know if it makes sense. The method doc/name may also need updation in that case.

Reply:
Done